tests/testAllObjects.py

```diff
@@ -47,6 +48,8 @@ class TestCat(InstanceCatalog):
 
 class basicAccessTest(unittest.TestCase):
 
+    longMessage = True
```

Comment:
Since it seems that pep-8 was fixed in some of the places in this pull request. Shouldn't the variable names in this file be lower_case instead of camelCase?